tensorboard/plugins/projector/projector_plugin.py

```diff
@@ -250,7 +261,49 @@ def is_active(self):
     Returns:
       A boolean. Whether this plugin is active.
```

Comment:
It's clear that `is_active` returns whether the plugin is active. This docstring should indicate what that means. ("The projector plugin is active when…")